services/openvpn/bootstrap.go

```diff
@@ -40,19 +41,9 @@ func Bootstrap() {
 	)
 
 	market.RegisterPaymentMethodUnserializer(
-		dto_openvpn.PaymentMethodPerTime,
+		pingpong.DefaultPaymentMethod.Type,
```

Comment:
Seems like you're registering same payment method unserialised several times, while only one is needed

Reply:
removed redundant ones